NextcloudTalk/CallViewController.m

```diff
@@ -190,6 +193,11 @@ - (void)viewDidLoad
     
     [self.collectionView registerNib:[UINib nibWithNibName:kCallParticipantCellNibName bundle:nil] forCellWithReuseIdentifier:kCallParticipantCellIdentifier];
     
+    [self.collectionView registerClass:CustomHeaderView.self forSupplementaryViewOfKind:UICollectionElementKindSectionHeader withReuseIdentifier:@"HeaderCollectionReusableView"];
+    
+    UICollectionViewFlowLayout *layout = self.collectionView.collectionViewLayout;
```

Comment:
This shows a warning
Could be cast as:
UICollectionViewFlowLayout *flow = (UICollectionViewFlowLayout *) self.collectionView.collectionViewLayout;